session/SessionHandler.php

```diff
@@ -29,9 +28,8 @@ public function close();
      * The return value (usually TRUE on success, FALSE on failure).
      * Note this value is returned internally to PHP for processing.
      * </p>
-     * @since 5.4
```

Comment:
I am not to much fan of removing the `since` tags, it always helps building old compat code